kanzucode/angie-client
# Ref-021 As an MC leader, I can update the number of salvations in my MC

**What does this PR do?**

- This PR adds a feature that allows MC Leaders to update information about salvations in a group event.

**Description of tasks?**

- MC Leaders can now update information about salvations recorded from a group event.

**How can I test this manually?**

- Create an event category called `Evangelism`
- Create a new group event under a group that you are a leader of, and set the event category to `Evangelism`.
- On the details page of the group event, there will be a tab for `Extra Data`.
- Edit the information under `Extra Data` and click `Submit`.
- When page is reloaded, changes submitted should be visible.

**Screenshot(s)**
![image](https://user-images.githubusercontent.com/68650343/110470674-308b3a00-80ec-11eb-8f47-c98e019bf8ee.png)
